### PR TITLE
{bio}[iccifort/2019.1.144-GCC-8.2.0-2.31.1] Trinity v2.8.5, Bowtie v1.2.2, ant v1.10.6

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@Micket did you forget the `Bowtie2` easyconfig?

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
nic425 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/75083cbb489d5aff95fcd894edaf31ed for a full test report.

**Comment by Micket** [comment]
@smoors Not sure how I managed to sneak in the wrong checksum.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
nic427 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/a6512a18a5c8a99377db6da91adde70b for a full test report.

File: easybuild/easyconfigs/a/ant/ant-1.10.6-Java-1.8.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'PackedBinary'
+
+name = 'ant'
+version = '1.10.6'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://ant.apache.org/'
```
Comment (smoors) [inline easybuild/easyconfigs/a/ant/ant-1.10.6-Java-1.8.eb]: please use https?

File: easybuild/easyconfigs/a/ant/ant-1.10.6-Java-1.8.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+description = """Apache Ant is a Java library and command-line tool whose mission is to drive processes described in
+ build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of
+ Java applications."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://apache.cu.be//ant/binaries']
```
Comment (smoors) [inline easybuild/easyconfigs/a/ant/ant-1.10.6-Java-1.8.eb]: please use https?

File: easybuild/easyconfigs/b/Bowtie/Bowtie-1.2.2-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+homepage = 'http://bowtie-bio.sourceforge.net/index.shtml'
+description = """Bowtie is an ultrafast, memory-efficient short read aligner.
+ It aligns short DNA sequences (reads) to the human genome."""
+
+toolchain = {'name': 'iccifort', 'version': '2019.1.144-GCC-8.2.0-2.31.1'}
+toolchainopts = {'pic': True, 'cstd': 'c++11'}
+
+sources = ['%(namelower)s-%(version)s-src.zip']
+source_urls = ['http://download.sourceforge.net/bowtie-bio/']
```
Comment (smoors) [inline easybuild/easyconfigs/b/Bowtie/Bowtie-1.2.2-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb]: please use https?

File: easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.3.5.1-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+description = """ Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads
+ to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s
+ of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes.
+ Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome,
+ its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes."""
+
+toolchain = {'name': 'iccifort', 'version': '2019.1.144-GCC-8.2.0-2.31.1'}
+toolchainopts = {'pic': True, 'cstd': 'gnu++98'}
+
+source_urls = [('http://sourceforge.net/projects/bowtie-bio/files/%(namelower)s/%(version)s', 'download')]
```
Comment (smoors) [inline easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.3.5.1-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb]: please use https?

File: easybuild/easyconfigs/t/Trinity/Trinity-2.8.5-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb [hunk 5]
```diff
@@ -0,0 +1,37 @@
+name = 'Trinity'
+version = '2.8.5'
+
+homepage = 'http://trinityrnaseq.github.io'
```
Comment (smoors) [inline easybuild/easyconfigs/t/Trinity/Trinity-2.8.5-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb]: please use https?

**Comment by Micket** [comment]
Bowtie 1.2.3 is already out so I will try to switch to it instead.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
nic426 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/a32602de7c786097e0309c69b1e3f46a for a full test report.

**Comment by smoors** [comment]
Going in, thanks @Micket!

**Pull request merged**